javascript/webapp/src/profiles/TestApp/TestApp.tsx

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { RecoilBasedProfileHost } from '@datashaper/app-framework'
+import { DefaultButton } from '@fluentui/react'
+import { memo, useCallback } from 'react'
+import type { MutableSnapshot, Snapshot } from 'recoil'
+import { atom, useRecoilState } from 'recoil'
+
+import type { TestAppResource } from './TestAppResource.js'
+
+export const TestApp: React.FC<{ resource: TestAppResource }> = memo(
```

Comment:
One way to use this as a plugin registration test but make it useful to the DS app could be to put help content or something here and then name it "Help" or "Guidance" or "Getting started" or something